src/lib/expansion/expansion-panel.ts

```diff
@@ -72,8 +72,8 @@ export const EXPANSION_PANEL_ANIMATION_TIMING = '225ms cubic-bezier(0.4,0.0,0.2,
   ],
 })
 export class MdExpansionPanel extends AccordionItem implements OnChanges, OnDestroy {
-  /** Whether the toggle indicator should be hidden. */
-  @Input() hideToggle: boolean = false;
+  /** The positioning of the expansion indicator. */
+  @Input() togglePosition: string = 'end';
```

Comment:
The type would be `'start' | 'end' | 'hidden'`

@crisbeto @kara what do you think about this API vs. having both `togglePosition` (only start/end) and `hideToggle`?

Reply:
I think I prefer the single `togglePosition` since it avoids bloating the API for a somewhat minor feature. It's also consistent with other inputs like the `MdFormField.floatPlaceholder`  which is `'always' | 'never' | 'auto'`.